values.yaml

```diff
@@ -232,6 +228,13 @@ global:
         memory: "50Mi"
         cpu: "20m"
 
+  # imageEnvoy is the name (and tag) of the Envoy Docker image used for the
+  # connect-injected sidecar proxies and the mesh, terminating, and ingress gateways.
+  #. If using Consul 1.7+, the envoy version must be 1.13+.
+  # If not set, the image used depends on the consul-k8s version. For
+  # consul-k8s 0.12.0 the default is envoyproxy/envoy-alpine:v1.13.0.
```

Comment:
Reminder to delete this comment since not setting this isn't supported.

Reply:
```suggestion
```